CONTRIBUTING.md

```diff
@@ -2,88 +2,156 @@
 
 Thank you for your interest in contributing to the Metaplex Aura Network! This document outlines our development process and guidelines.
 
+## Getting Started
+
+### Prerequisites
+
+-   **Rust:** This project requires Rust **version 1.84**. We recommend using `rustup` to manage your Rust installation. You can install `rustup` by following the instructions at [https://rustup.rs/](https://rustup.rs/). After installing `rustup`, ensure you have the correct version:
```

Comment:
tbh i believe we needn't specify the rust version, as it becomes pretty evident from `rust-toolchain.toml`.

Reply:
I was trying to approach this as if I knew nothing about Rust (which is not that far from truth 😄) and provide a guideline of what should I do. Is this too much of specifics?

Reply:
i think it's fine if we leave out the version; the rest in this section is good